x/epochs/abci_test.go

```diff
@@ -0,0 +1,114 @@
+package epochs_test
+
+import (
+	"testing"
+	"time"
+
+	simapp "github.com/c-osmosis/osmosis/app"
+	"github.com/c-osmosis/osmosis/x/epochs"
+	"github.com/c-osmosis/osmosis/x/epochs/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestEpochInfoChangesBeginEndBlockersAndInitGenesis(t *testing.T) {
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+
+	app.EpochsKeeper.DeleteEpochInfo(ctx, "daily")
+	app.EpochsKeeper.DeleteEpochInfo(ctx, "weekly")
```

Comment:
Why are these initially deleted?

Reply:
It is deleted to test init genesis from fresh status on following lines

Reply:
oh because simapp uses the Osmosis default. That makes sense, can a comment be added here that this must be synced with what is in app/app.go, and make a TODO: for making something more modular here post-launch

Reply:
Updated